crates/sozo/ops/src/migration/migrate.rs

```diff
@@ -699,6 +699,9 @@ where
     P: Provider + Sync + Send + 'static,
 {
     ui.print("\n📋 Migration Strategy\n");
+    if let Some(world) = &strategy.world {
+        ui.print_sub(format!("declare (contract address: {:#x})\n", world.contract_address));
+    }
```

Comment:
This should be placed into the already existing test on world strategy (few lines below).

Reply:
i suggested to put it outside because we world contract address is always present regardless of if we need to migrate the world or not